README.md

```diff
@@ -35,6 +35,10 @@ Follow this file : [WindowsMSys2_InstallGuide.md](https://github.com/RetroShare/
 
 Follow this file : [MacOS_X_InstallGuide](https://github.com/RetroShare/RetroShare/blob/master/build_scripts/OSX/MacOS_X_InstallGuide.md)
 
+## Compilation on Linux
+
+Follow this file : [Linux_InstallGuide](https://github.com/defnax/RetroShare/blob/master/build_scripts/Debian%2BUbuntu/Linux_InstallGuide.md)
```

Comment:
Seems weird that you refer to a file in your own fork. Is that a mistake?

Reply:
oh, seems a mistake, corrected it